Tests/SwiftStdlibTests/ArrayExtensionsTests.swift

```diff
@@ -11,6 +11,12 @@
 import XCTest
 
 final class ArrayExtensionsTests: XCTestCase {
+    func testInitClosure() {
+        let squares = [0, 1, 4, 9, 16, 25, 36, 49, 64, 81, 100]
+        let array = [Int].init(numberOfElements: 11) { $0*$0 }
```

Comment:
Does it work like this?
```suggestion
        let array = [Int](numberOfElements: 11) { $0 * $0 }
```

Reply:
fixed